extensions/vertx-http/deployment/src/main/resources/dev-templates/tags/nonActionableExtension.html

```diff
@@ -2,10 +2,16 @@
 <div class="col mb-4">
     <div class="card h-100">
         <div class="card-header">
-            {it['name']}
-            {#if it['metadata'] && it['metadata']['guide']}
-            <a href="{it['metadata']['guide']}" class="text-muted float-right" title="Guide"><i class="fa fa-book"></i></a>
-            {/if}
+            <div class="row">
+                <div class="col-md-10">
+                    {it['name']}
+                </div>
+                <div class="col-md-1">
```

Comment:
From what I can see, the icon is significantly more to the left. Could we adjust that with some alignment and padding?

Reply:
padding done

![after-style](https://user-images.githubusercontent.com/10883521/117165726-227c3180-ad9c-11eb-9b2a-29acc224e522.png)